docs/installation/development-rack/ubuntu.md

```diff
@@ -8,131 +8,49 @@ url: /installation/development-rack/ubuntu
-
-### Installation Workaround
-
-In order to get the kubernetes steps that follow to work, a slight modification to your local `/etc/resolv.conf` file is required. `microk8s` leverages your local DNS configuration to configure DNS resolution in Kubernetes, and the following commands will provide you with a workaround to the problem described above:
-
-Make a backup of your `/etc/resolv.conf` file:
-```html
-    $ mv /etc/resolv.conf /etc/resolv.conf.orig
-```
-Important: Create a new configuration file by removing the `trust-ad` option from the options line in the original configuration file:
-```html
-    $ cat /etc/resolv.conf.orig | sed 's/trust-ad//g' > /etc/resolv.conf.manually-configured
-```
-Next, create a symbolic link to the newly created `/etc/resolv.conf.manually-configured` file:
-```html
-    $ ln -s /etc/resolv.conf.manually-configured /etc/resolv.conf
-```
-These steps allow your local DNS resolver to be manually configured.  Finally, restart the following processes to have this modification take affect:
-```html
-    $ systemctl restart daemon-reload
-    $ systemctl restart systemd-networkd systemd-resolved
-```
-Now, the Kubernetes steps below should work as expected.
-
----
-
-### Kubernetes
-```html
-    $ snap install microk8s --classic --channel=1.13/stable
-    $ microk8s.start
-    $ microk8s.enable dns storage
-    $ mkdir -p ~/.kube
-    $ microk8s.config > ~/.kube/config
-    $ sudo snap restart microk8s
-    $ sudo snap alias microk8s.kubectl kubectl
-```
-### Terraform
+## Installation
 
-- Install [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html)
+Convox development racks run on Minikube with several required options: 
+- `--insecure-registry=”registry.<RACK_NAME>.localdev.convox.cloud”`
+- `--kubernetes-version=<k8s_VERSION>` 
+- `--static-ip <IP>` 
 
-### Convox CLI
+The first thing you must choose is the name for your rack as it will be a component of other installation parameters.  For this documentation we will use the rack name `test-local` 
 
-### x86_64 / amd64
+Then verify the Kubernetes version for the [rack version](https://github.com/convox/convox/releases) you are going to install.  At the time of this writing it is `v1.23.0` 
 
-```html
-    $ curl -L https://github.com/convox/convox/releases/latest/download/convox-linux -o /tmp/convox
-    $ sudo mv /tmp/convox /usr/local/bin/convox
-    $ sudo chmod 755 /usr/local/bin/convox
-```
+Finally choose a static IP that works for your local configuration.  We have found `192.168.212.2` to work well in most all cases. 
```

Comment:
different ip will not work since, we are mapping this to dns entry

Reply:
got it